src/models/requestData.js

```diff
@@ -1,9 +1,29 @@
 import getVerboseColumns from '../utils/getVerboseColumns.js'
 import logger from '../utils/logger.js'
+import axios from 'axios'
+import config from '../../config/index.js'
 
 export default class RequestData {
```

Comment:
Maybe this is now inappropriately named? Seems like it wraps the HTTP response for an async request and facilitates the fetching of the response details.  Might be better to split out the request model and response details models?

Reply:
we can pull this out into another ticket for now